packages/react-native-payments/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-native-payments",
+  "version": "0.0.0",
+  "description": "Payments solutions for react-native apps.",
+  "main": "src/index.js",
+  "files": [
+    "src"
+  ],
+  "license": "UNLICENSED",
+  "publishConfig": {
```

Comment:
Just to keep consistency across all packages, please move `publishConfig` to the end and `peerDependencies` after `devDependencies` 😇

Reply:
Should be ok now, please double check 👍

Reply:
You just missed moving `peerDependencies` below `devDependencies` 😬

Reply:
Sorry about that, missed it. Should be ok now 👍